runtime/syntax/buzz.yaml

```
@@ -0,0 +1,53 @@
+filetype: buzz
+
+detect:
+    filename: "\\.buzz$"
+
+rules:
+    # keywords
+    - keywords: "\\b(range|namespace|typeof|type|zdef|void|static|var|const|extern|import|export|from|test|as|in|while|if|else|fun|try|catch|throw|constructor|return|switch|default|break|for|foreach|do|until|continue|resolve|resume|yield|any|out)\\b"
+      # special method names
+    - symbol.operator: "\\b(!|and|or|band|bor|xor|\\?\\?)\\b"
+      # definitions
+    - identifier: "def [a-zA-Z_0-9]+"
+      # keywords (declaration)
+    - statements: "\\b(enum|object|protocol)\\b"
+      # constants (true, false, null, this)
+    - constants: "\\b(true|false|null|this)\\b"
+      # constants (types)
+    - type: "\\b(ud|str|int|float|obj|fib|bool|pat|type|any)\\b\\??"
+      # decorators
+    - preproc: "^\\s*@[^(]*"
+      # operators
+    - symbol.operator: "(=|\\+=|\\-=|\\/=|\\*=)"
+      # parentheses
+    - symbol.brackets: "([(){}]|\\[|\\])"
+      # extra operators
+    - symbol.operator: "(\\+|\\-|\\/|%|\\*|>>|<<|\\\\|\\^|&|~|!>)"
+      # Comparision operators
+    - symbol.operator: "(==|!=|<=?|>=?|\\bis\\b|\\?\\?)"
+      # numbers
+    - constant.number: "\\b[0-9](_?[0-9])*(\\.([0-9](_?[0-9])*)?)?(e[0-9](_?[0-9])*)?\\b" # decimal
+    - constant.number: "\\b0b(_?[01])+\\b"     # bin
+    - constant.number: "\\b0o(_?[0-7])+\\b"    # oct
+    - constant.number: "\\b0x(_?[0-9a-fA-F])+\\b" # hex
+
+    - constant.string:
+        start: "\""
+        end: "(\"|$)"
+        skip: "\\\\."
+        rules:
+            - constant.specialChar: "\\\\."
+
+    - constant.string:
+        start: "'"
+        end: "('|$)"
+        skip: "\\\\."
+        rules:
+            - constant.specialChar: "\\\\."
+
+    - comment:
+        start: "\\|"
+        end: "$"
+        rules:  # AKA Code tags (PEP 350)
+            - todo: "(TODO|FIXME|HACK|BUG|NOTE|FAQ|MNEMONIC|REQ|RFE|IDEA|PORT|\\?\\?\\?|!!!|GLOSS|SEE|TODOC|STAT|RVD|CRED):?"
```

I'm not sure if these are even relevant in Buzz but the comment about PEP is certainly Python-specific.

```suggestion
        rules:
            - todo: "(TODO|FIXME|HACK|BUG|NOTE|FAQ|MNEMONIC|REQ|RFE|IDEA|PORT|\\?\\?\\?|!!!|GLOSS|SEE|TODOC|STAT|RVD|CRED):?"
```